<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorColorSchemeWidget</class>
 <widget class="QWidget" name="EditorColorSchemeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>556</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPatchedComboBox" name="cbScheme"/>
      </item>
      <item>
       <widget class="QToolButton" name="btnSchemeMenu">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>7</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeView" name="treeItems">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>7</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widgetSchemeItem" native="true">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="cbBackground">
                  <property name="text">
                   <string>Background:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="cbForeground">
                  <property name="text">
                   <string>Foreground:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="ColorEdit" name="colorForeground">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="0" column="2">
                 <widget class="ColorEdit" name="colorBackground">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QGroupBox" name="grpFontStyles">
                  <property name="title">
                   <string>Font Styles</string>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <item>
                    <widget class="QCheckBox" name="cbBold">
                     <property name="text">
                      <string>Bold</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="cbItalic">
                     <property name="text">
                      <string>Italic</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="cbStrikeout">
                     <property name="text">
                      <string>Strikeout</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="cbUnderlined">
                     <property name="text">
                      <string>Underlined</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="Editor" name="editDemo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRainborParenthesis">
     <property name="text">
      <string>Rainbow parenthesis</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionCopy_Scheme">
   <property name="text">
    <string>Duplicate...</string>
   </property>
  </action>
  <action name="actionRename_Scheme">
   <property name="text">
    <string>Rename...</string>
   </property>
  </action>
  <action name="actionReset_Scheme">
   <property name="text">
    <string>Restore to Default</string>
   </property>
  </action>
  <action name="actionImport_Scheme">
   <property name="text">
    <string>Import Scheme...</string>
   </property>
  </action>
  <action name="actionExport_Scheme">
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionDelete_Scheme">
   <property name="text">
    <string>Delete...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Editor</class>
   <extends>QFrame</extends>
   <header>editor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorEdit</class>
   <extends>QFrame</extends>
   <header location="global">widgets/coloredit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPatchedComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widgets/qpatchedcombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbScheme</tabstop>
  <tabstop>btnSchemeMenu</tabstop>
  <tabstop>treeItems</tabstop>
  <tabstop>cbBackground</tabstop>
  <tabstop>cbForeground</tabstop>
  <tabstop>cbBold</tabstop>
  <tabstop>cbItalic</tabstop>
  <tabstop>cbStrikeout</tabstop>
  <tabstop>cbUnderlined</tabstop>
  <tabstop>chkRainborParenthesis</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
